_posts/2020-11-04-differences-between-programming-languages.md

```diff
@@ -0,0 +1,169 @@
+
+C++ was developed based upon on C Language. C++ is more powerful and it is a high-performance language but a low-level programming language. As a low-level programming language, it is relatively complicated and it is pretty hard to learn. C++ requires tons of code in order to function properly. [^8] The main difference between C and C++ is that C is procedural programming language and C++ is a combination of procedural and object-oriented programming language. [^9]
+
+### Keywords
+- Keywords in C and C++: auto, break, char, double, float, do, else, void, while, switch, register...
+- Keywords that were added into C++: bool, catch, class, false, const_cast, try, public, using, this, private...
+
+## 7. SQL
+![SQL](http://tugrul.dbsdataprojects.com/wp-content/uploads/sites/162/2017/03/Introduction-to-SQL.png)
+
+SQL is a domain-specific language. It is used to manage data and for stream processing. It is particularly useful for dealing with structured data. SQL has multiple features in terms of data management, including data query, data manipulation, data storage and data access control. [^10]
+
+SQL is used by many companies which they need to gather data, such as UnitedHealthCare and airline companies. In 1987, SQL was used as a standard of the International Organization for Standardization (ISO). [^10]
+
+
+# Comparison Between Different Languages
+| Languages | Features | Best For | Used By | Easy to Learn
+| --- | --------------- | --------------- | ---------------- | ----- |
+| Python | multi-paradigm, simple, high readability | machine learning, data analysis, web design | Google, Pinterest, Youtube, NASA | ⭐️⭐️⭐️⭐️⭐️|
+| Java | interpreted language, extensive network library | web development | Android apps | ⭐️⭐️|
+| JavaScript | flexible, new front-end frameworks | front-end website, video games | Almost every website | ⭐️⭐️⭐️⭐️ |
+| Ruby | simple and flexible, completely object-oriented | web development，web applications, security | Amazon | ⭐️⭐️⭐️⭐️ |
+| Bash | brace expansion, command line completion, extremely useful in all IT works | IT professions | Everyone | ⭐️⭐️⭐️⭐️|
+| HTML | easy to use, multiple website formatting versions | web developers | Apple, CyberCoders | ⭐️⭐️⭐️|
+| C | can be used in almost all systems | operating systems, software development| Apple, Microsoft | ⭐️|
+| C++ | quick processing, widely-used in software | operating systems, video games, application and software development | Google, Firefox, Blackberry OS | ⭐️⭐️ |
+| SQL | free and easily accessible, simple syntax | data processing, database management | nearly all applications | ⭐️⭐️⭐️ |
+
+
```

Comment:
I recommend adding a header to the bibliography section `# Bibliography`

Reply:
Hi @melt6457 Thanks for the advice! I have added a header and changed a new image. Please take a look and let me know if I need to change anything else.